server/src/main/java/org/kaazing/gateway/server/config/parse/translate/November2015Validator.java

```diff
@@ -15,6 +15,7 @@
  */
 package org.kaazing.gateway.server.config.parse.translate;
 
+import org.kaazing.gateway.server.config.parse.GatewayConfigNamespace;
```

Comment:
Should this class be renamed to November2015ToWhateverLatestNamesSpaceIsTranslator, that seems like the pattern in the file name below